assets/prefabs/culture.prefab

```diff
@@ -0,0 +1,9 @@
+{
+  "Culture" : {
+    "name" : "Hjerl Hede",
+    "buildingNeedPerZone" : {"RESIDENTIAL" : 1, "COMMERCIAL" : 3},
```

Comment:
The school is `GOVERNMENTAL` and the windmill is `AGRICULTURE` - should those zones be mentioned in here?